src/eei.h

```diff
@@ -108,8 +108,18 @@ struct EthereumInterface : ShellExternalInterface {
 };
 
 struct GasSchedule {
+  static constexpr unsigned sLoad = 50;
   static constexpr unsigned storageStoreCreate = 20000;
   static constexpr unsigned storageStoreChange = 5000;
+  static constexpr unsigned log = 375;
+  static constexpr unsigned logData = 8;
+  static constexpr unsigned logTopic = 375;
+  static constexpr unsigned create = 32000;
+  static constexpr unsigned call = 40;
+  static constexpr unsigned memory = 3;
+  static constexpr unsigned copy = 3;
```

Comment:
What are these two used for?

Reply:
They are used in codecopy and calldatacopy, according to outdated yellowpaper semantics. Will fix this

Reply:
Can you name perhaps have `codeMemory`, `codeCopy` and `callDataMemory`, `callDataCopy`?

Reply:
yeah, I was using the fee schedule names from EVMJIT and the yellowpaper but this makes more sense i guess.